list.go

```diff
@@ -1,42 +1,13 @@
 package certificatetpr
 
 import (
-	"encoding/json"
-
-	"k8s.io/client-go/pkg/api/unversioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // List represents a list of CustomObject resources.
 type List struct {
-	unversioned.TypeMeta `json:",inline"`
-	Metadata             unversioned.ListMeta `json:"metadata" yaml:"metadata"`
+	metav1.ObjectMeta `json:"metadata"`
+	metav1.TypeMeta   `json:",inline"`
```

Comment:
Why should we remove the YAML tags? AFAIK they have been quite important because of some decoding issues.